### DISCUSSION TITLE
ROMs

### DISCUSSION THREAD

**Discussion by curif** [curif/AgeOfJoy-2022.1#83] [General]
If you've tried a ROM, can you write a message in this thread with the results?

```
ROM:
MD5SUM:
is working?
```
First check here: https://curif.notion.site/AGE-of-Joy-Rom-compatibility-Database-a3793cebeb5c4a7f9b1cd2c3306d2ee1
MD5 calculator: https://curif.github.io/AgeOfJoy-ROMCRC/index.html

**Comment by Elleby77** [comment]
ROM: ddragon.zip
MD5SUM: ecde6adb3068003f41a31f6a913c8e38
is working? Yes

I tried Double Dragon and it worked fine.  Only issue is that the punch button is on the left controller.    Also once moved into the actual arcade there seems to be a bit of slowdown.

**Comment by Elleby77** [comment]
ROM: popeye.zip
MD5SUM: 73bb8b7c4823ca4b7e3792fef5d64c15
is working? Yes

**Comment by Elleby77** [comment]
ROM: dkongjr.zip
MD5SUM: 2b554f428c1fb9bca4f9f10d2abed5b4
is working? Yes

**Comment by WilsonBradley** [comment]
Thank you so much...

Sincerely,

Wilson Bradley
***@***.***

**Comment by Elleby77** [comment]
ROM: dkong3.zip
MD5SUM: 53bcbf8591f68d360f36cda8ad766d85
is working? Yes

ROM: mario.zip
MD5SUM: 7c46584285d89224afeaaa25829d2491
is working? Yes

ROM: excitebk.zip
MD5SUM: 61e3f658440a5b58fd730c7d53b2c4fc
is working? Yes

ROM: suprmrio.zip
MD5SUM: 62a65e41f80c754ced3d6f2d4dde8d19
is working? Yes

ROM: drmario.zip
MD5SUM: 02a0dae6c9a11454a5970acc70207573
is working? Yes

ROM: radarscp.zip
MD5SUM: 5a6a1e4c68d6c232b0e83a12444f8f41
is working? Yes

ROM: starwars.zip
MD5SUM: 70053211920793b0a7eedc00defc34e6
is working? No :(  Screen is not visible, but you can hear sound maybe because its a vector game?

**Reply by curif** [reply]
Amazing!
starwars isn't working, sadly, it's a game loved by a lot of people. May be uses an image format that is not supported yet.
I already registered your tests  in the DB: https://www.notion.so/curif/AGE-of-Joy-Rom-compatibility-Database-a3793cebeb5c4a7f9b1cd2c3306d2ee1

Thank you!!!

**Comment by WilsonBradley** [comment]
This is great news..
The library is growing.

Sincerely,

Wilson Bradley
***@***.***

**Comment by Elleby77** [comment]
ROM: tmnt2po.zip
MD5SUM: d6e39fa62fbe886f58fc21dc0311e812
is working? Yes, I chose this version so you are allowed to select your turtle at the start

ROM: superman.zip
MD5SUM: f003920c110baedfd5f212f87d5f85c2
is working? Yes

**Comment by Elleby77** [comment]
ROM: cninja.zip
MD5SUM: 00009fcefa24c6f06e69922da826456f
is working? Yes

ROM: ffight.zip
MD5SUM: 3b2c8f1924d343dbd6c79d75b4fdca37
is working? Yes

ROM: ghouls.zip
MD5SUM: 5d87c44a02a492c33ce348d13b1a10f4
is working? Yes

ROM: gng.zip
MD5SUM: b211b6a0cb639d9a188d2a18c5f5d8bb
is working? Yes

ROM: karnov.zip
MD5SUM: dd968025a3b3c4f0fdf0d1faa9609001
is working? Yes

ROM: kungfum.zip
MD5SUM: ad00239b09a9bf7f80aecf3822fef32f
is working? Yes

ROM: lbowling.zip
MD5SUM: c3760f868cf4aa129c70c4bb344a07f9
is working? Yes

ROM: maglord.zip
MD5SUM: f94e0611e25b206a255251d9a09ca30f
is working? Yes

ROM: tagteam.zip
MD5SUM: 8dcdf74986132f414f96a0b5184c71cf
is working? Yes

**Reply by curif** [reply]
👏
